### DISCUSSION TITLE
My repository is not appearing among the repositories of any topic

### DISCUSSION THREAD

**Discussion by hjk789** [community/community#23126] [New to GitHub]
<p>My repository (<a href="http://github.com/hjk789/NXEnhanced" class="inline-onebox" rel="noopener nofollow ugc">GitHub - hjk789/NXEnhanced: Adds "quality-of-life" features to NextDNS website for a more practical usability</a>) is not appearing in the list of repositories of any topic I’ve set. My other repositories are appearing normally. This one is the only one that doesn’t. Am I missing something? Is this a bug or what?</p>

**Comment by jfullstackdev** [comment]
<p>there is a similar post for this one and the reason why it’s not appearing in the topics it is listed is because the repo is not active anymore for a long time,</p>
<p>the algo for that, as we conclude, it’s maximizing those active repos listed in a certain topic</p>

**Comment by hjk789** [comment]
<p>Then I think there’s a bug, because in my other repo (<a href="https://github.com/hjk789/VPNsManager" class="inline-onebox" rel="noopener nofollow ugc">GitHub - hjk789/VPNsManager: Easily manage client SoftEther VPN Gate servers, fetch new ones, ping and view max speed</a>) the last commit is much older than this one, yet it appears normally.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>yup you can always file a bug here</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github-community/community">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github-community/community" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181074991-2ff0c7eb-7e0d-494d-b6bd-eb799f41145d.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/5/65ba3e185fa42d1b05ddfd60475ed707f0a4e195_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/5/65ba3e185fa42d1b05ddfd60475ed707f0a4e195_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/5/65ba3e185fa42d1b05ddfd60475ed707f0a4e195.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/5/65ba3e185fa42d1b05ddfd60475ed707f0a4e195_2_10x10.png"></div>

<h3><a href="https://github.com/github-community/community" target="_blank" rel="noopener">GitHub - github-community/community: Public feedback discussions for: GitHub...</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - GitHub - github-community/community: Public feedback discussion...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>I can’t remember the other one whether the user file a bug</p>

**Comment by hjk789** [comment]
<p>I filed a bug, they quickly re-indexed my repository. It’s fixed now.</p>
